docs/websites/lamp/lamp-server-on-ubuntu-14-04-trusty.md

```diff
@@ -0,0 +1,298 @@
+modified_by:
+  name: Joel Kruger
+published: 'Monday, February 23rd, 2015'
+title: LAMP Server on Ubuntu 14.04 LTS
+---
+
+This guide provides step by step instructions for installing a full featured LAMP stack on an Ubuntu 14.04 (Trusty Tahr) system. The 14.04 release is the most current LTS (long term support) version of the Ubuntu operating system, and will be supported with security updates until April of 2019.
+
+In this guide, you'll learn how to set up Apache, MySQL, and PHP. If you don't need MySQL or PHP, please don't feel obligated to install them.
+
+ {: .note }
+> For this LAMP guide it was decided that using PHP-FPM is preferred over prefork MPM for performance benefits.
+ 
+ {: .note }
+> Throughout this guide we will offer several suggested values for specific configuration settings. Some of these values will be set by default. These settings are shown in the guide as a reference, in the event that you change these settings to suit your needs and then need to change them back.
+
+## Prerequisites
+
+Your Linode should already be set up according to the instructions in our [Getting Started](/docs/getting-started) guide.
+
+As a friendly reminder, since you will be exposing your server (and content) to the rest of the world, it is an excellent choice to implement some security precautions. For assistance with this, please see our documentation: [Securing Your Server](https://linode.com/docs/security/securing-your-server)
+
+{: .note}
+>This guide is written for a non-root user. Commands that require elevated privileges are prefixed with ``sudo``. If you're not familiar with the ``sudo`` command, you can check our [Users and Groups](/docs/tools-reference/linux-users-and-groups) guide.
+
+
+Set the Hostname
+----------------
+
+Before you begin installing and configuring the components described in this guide, please make sure you've followed our instructions for [setting your hostname](/docs/getting-started#sph_setting-the-hostname). Issue the following commands to make sure it is set properly:
+
+    hostname
+    hostname -f
+
+The first command should show your short hostname, and the second should show your fully qualified domain name (FQDN).
+
+Install and Configure the Apache Web Server
+-------------------------------------------
+
+The Apache web server is a popular choice for serving web pages. While many alternatives have appeared in the last few years, Apache remains a powerful option that we recommend for most uses.
+
+1. Make sure your package repositories and installed programs are up to date by issuing the following commands:
+
+        sudo apt update
+        sudo apt upgrade 
+
+2. To install the current version of the Apache web server (in the 2.x series) on an Ubuntu system use the following command:
+
+        sudo apt install apache2
+
+3. Edit the main Apache configuration file to adjust the resource use settings.
+
+{: .file }
+/etc/apache2/apache2.conf
+:   ~~~ apache
+    KeepAlive On
```

Comment:
`On` is [default](https://httpd.apache.org/docs/2.4/mod/core.html#keepalive). I suggest removing this section entirely.